backend/app/api/ws/chat_namespace.py

```
@@ -222,24 +239,22 @@ async def on_connect(self, sid: str, environ: dict, auth: Optional[dict] = None)
             logger.warning(f"[WS] Missing token in auth sid={sid}")
             raise ConnectionRefusedError("Missing authentication token")
 
-        # Verify token
-        user = verify_jwt_token(token)
+        # Verify token using unified verification (supports both JWT and API Key)
+        user, token_exp, auth_type = verify_websocket_token(token)
         if not user:
             logger.warning(f"[WS] Invalid token sid={sid}")
             raise ConnectionRefusedError("Invalid or expired token")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)